distribution/docker/solr-docker/src/main/docker/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN unzip /jars/solr-schema-${project.version}.jar -d /prep
 # Final Image Stage
 FROM solr:${solr.version}
 # Copy solr libs
-COPY maven/jts-core-1.15.0.jar /opt/solr/server/solr-webapp/webapp/WEB-INF/lib/
+COPY maven/jts-core-${jts.spatial4j.version}.jar /opt/solr/server/solr-webapp/webapp/WEB-INF/lib/
```

Comment:
❓ Just want to ensure I'm understanding this correctly. The docker-maven-plugin is handling the interpolation of these props right?

Reply:
That is correct.